roboteam_ai/src/utilities/OffensiveCoach.cpp

```diff
@@ -0,0 +1,312 @@
+//
+// Created by robzelluf on 3/8/19.
+//
+
+#include <roboteam_ai/src/interface/widget.h>
+#include <roboteam_ai/src/interface/drawer.h>
+#include "OffensiveCoach.h"
+
+namespace rtt {
+namespace ai {
+namespace coach {
+
+int OffensiveCoach::maxPositions = 4;
+double OffensiveCoach::maxDistanceFromBall = 6.0;
```

Comment:
magic numbers

Reply:
https://cdn.drawception.com/images/panels/2018/5-14/EC5N43PsAP-2.png

Reply:
But as mentioned before here, since these are very specific to the OffensiveCoach I am not sure whether it makes a lot of sense to put them in constants. They are not used anywhere else and they're still specified in the beginning of the file

Reply:
Yeah true, but needed, and too specific to be moved to constants